charts/newrelic-logging/values.yaml

```diff
@@ -284,3 +284,12 @@ windows:
 
 # -- Sets pod dnsConfig. Can also be configured with `global.dnsConfig`
 dnsConfig: {}
+
+# -- Use persistent volumes to store the fluent-bit db, this will override `fluentBit.db` path and use `/db/${NODE_NAME}-fb.db` file instead
+persistentVolume:
+  enabled: false
+  size: 10Gi
+  # -- The storage class should allow ReadWriteMany mode
+  storageClassName: "standard-rwx"
```

Comment:
Is this cloud specific? AFAIK each cloud provider have its own volume provisioning solution... We maybe could even skip the PV completely and use a storage class (cloud specific) that can dynamically provide (create) a PV for a given PVC 🤔

Reply:
I'm looking now at Local Persistent Volumes, AFAIK the use of ReadWriteMany is mandatory in this case as this will only create a volume for all nodes (daemonsets and deployments share the same volumes, statefulset create a different one per instance)

Reply:
Sure, this one is for GKE, each cloud provider will provide their own storage classes but those should be configured manually, or at least IDK how to get the one we're interested (RWX) without setting it directly. The option to change it should be available because maybe someone want to use this in a custom kubernetes cluster or so and they need to set the PV (that's why I'm adding the dynamicProvisioning flag)